### DISCUSSION TITLE
How to use lego

### DISCUSSION THREAD

**Discussion by vadlike** [AdguardTeam/AdGuardHome#3134] [Q&A]
i have read this manual. https://github.com/AdguardTeam/AdGuardHome/wiki/Encryption#lego i have got finialy 2 files .key not .pe. like using certbot. but i tryed to chose this files .key like .pem it's not working. i need correct manual please

**Comment by vadlike** [comment]
> i have read this manual. https://github.com/AdguardTeam/AdGuardHome/wiki/Encryption#lego i have got finialy 2 files .key not .pem like using certbot. but i tryed to chose this files .key like .pem it's not working. i need correct manual please

**Reply by ameshkov** [reply]
Once you got them, use them as per the instruction: https://github.com/AdguardTeam/AdGuardHome/wiki/Encryption#configure-adguard-home

.key file contents goes to the key field, .pem to the cert field.
